config/hiveadmission/clusterdeployment-webhook.yaml

```diff
@@ -1,10 +1,13 @@
 ---
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
   name: clusterdeploymentvalidators.admission.hive.openshift.io
 webhooks:
 - name: clusterdeploymentvalidators.admission.hive.openshift.io
+  admissionReviewVersions:
+  - v1
+  - v1beta1
```

Comment:
How does registering both v1beta1 and v1 works with our present webhook code?

Reply:
I had thought this was a list of versions to *try*, in order, and it would accept the first one that worked. But it's apparently not that simple. I'm removing v1 from these.